playwright-tests/lib/fetchHelper.ts

```diff
@@ -0,0 +1,21 @@
+import {Frame, Page} from '@playwright/test';
```

Comment:
The files under `playwright-tests/lib/` are roughly equivalent to the helpers under https://github.com/GoogleChrome/workbox/tree/v6/infra/testing